README.md

```diff
@@ -1,56 +1,40 @@
 ![](https://img.shields.io/badge/Microverse-blueviolet)
 
-# Project Name
-
-> Description the project.
+# Coffee Summit 2022
 
+> First capstone project in Microverse. In this project i've build an online website for an imaginary conference. 
 
 ## Built With
 
-- Major languages
-- Frameworks
-- Technologies used
+- HTML
+- CSS
+- JavaScript
 
 ## Live Demo (if available)
 
-[Live Demo Link](https://livedemo.com)
-
+[Live Demo Link](https://karayamanemre.github.io/coffee-summit/)
```

Comment:
- [ ] Live link is not working yet, kindly update it since it is a project requirement

Reply:
it will work after i merge this branch. i set the main branch as source since i will delete this branch after merge.